Code Quality/Tools.md

```diff
@@ -162,15 +160,14 @@ Even though [TSLint](https://palantir.github.io/tslint/) is deprecated, it is st
 
 Infinum created various rulesets for ESLint and TSLint which you can check out [here](https://github.com/infinum/js-linters). We encourage you to use some of these presets, as per your needs:
 
-  - [Base ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config/README.md)
-  - [React ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config-react/README.md)
-  - [Angular TSLint config](https://github.com/infinum/js-linters/tree/master/tslint-config-angular)
+-   [Base ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config/README.md)
```

Comment:
Since you're already updating this chapter, could you also fix the links? Darac discovered yesterday that they don't work.

Reply:
Do we not have eslint setup for angular?